### PR TITLE
Aj union set volumes

### PR DESCRIPTION
**Pull request by ajimoreno**
The protocol "pwem - join sets" has been modified to take several Volumes as input and create a SetOfVolumes as output.

### CODE REVIEW

File: pwem/protocols/protocol_sets.py [hunk 1]
```diff
@@ -139,13 +140,19 @@ def createOutputStep(self):
 
         # Read ClassName and create the corresponding EMSet (SetOfParticles...)
         try:
-            outputSetFunction = getattr(self, "_create%s" % set1.getClassName())
+            if str(set1.getClassName()) is not 'Volume':
```
Comment (pconesa) [inline pwem/protocols/protocol_sets.py:143]: I now 'Volume' is in many places, but we should try to improve this having a single class name source.

I'd suggest to replace 'Volume' by  `Volume.__name__`

File: pwem/protocols/protocol_sets.py [hunk 2]
```diff
@@ -139,13 +140,19 @@ def createOutputStep(self):
...
+                outputSetFunction = getattr(self, "_create%s" % set1.getClassName())
+            else:
+                outputSetFunction = getattr(self, "_createSetOfVolumes")
```
Comment (pconesa) [inline pwem/protocols/protocol_sets.py:146]: Can't this be replaced by "self._createSetOfVolumes" ?

**Review by pconesa — CHANGES_REQUESTED** [review]
Thanks @ajimoreno 
I've got some minor changes to do.

File: pwem/protocols/protocol_sets.py (hunk 1 above)
Comment (ajimoreno) [reply pwem/protocols/protocol_sets.py]: Done

File: pwem/protocols/protocol_sets.py (hunk 2 above)
Comment (ajimoreno) [reply pwem/protocols/protocol_sets.py]: Done

Comment (pconesa) [reply pwem/protocols/protocol_sets.py]: Thanks @ajimoreno I've added a test for this case

**Review by pconesa — APPROVED** [review]

**Pull request merged**